packages/manager/modules/pci/src/projects/project/storages/databases/database/query-statistics/query-statistics.html

```diff
@@ -0,0 +1,83 @@
+<div class="container-fluid px-0 mt-3">
+    <!-- error messages -->
+    <cui-message-container
+        data-messages="$ctrl.messages"
+    ></cui-message-container>
+
+    <h2 data-translate="pci_databases_query_statistics_title"></h2>
+
+    <p data-translate="pci_databases_query_statistics_description"></p>
+    <oui-datagrid
+        data-rows="$ctrl.queryStatistics"
+        data-empty-placeholder="{{ 'pci_databases_query_statistics_retrieve_error' | translate }}"
```

Comment:
`data-empty-placeholder="{{ :: 'pci_databases_query_statistics_retrieve_error' | translate }}"`

Reply:
And for other column headers as well.